ambry-quota/src/main/java/com/github/ambry/quota/MaxThrottlePolicy.java

```diff
@@ -29,13 +29,11 @@ public ThrottlingRecommendation recommend(List<QuotaRecommendation> quotaRecomme
     boolean shouldThrottle = false;
     Map<QuotaName, Float> quotaUsagePercentage = new HashMap<>();
     int recommendedHttpStatus = 200;
-    Map<QuotaName, Double> requestCost = new HashMap<>();
```

Comment:
nit: please update the comment of this method, i.e. remove 
```
 Also groups the quota usage and request cost for all the quotas.
 ```

Reply:
fixed.